README.md

```diff
@@ -22,6 +24,41 @@ microservices.
 - all nodes are equal, no master/leader node
 
 
+## Getting Started
+### Install the Gem
+
+```ruby
+gem install "moleculer-ruby"
+```
+
+or add to your Gemfile:
+
+```ruby
+gem "moleculer-ruby", "~>0.1"
+```
+
+### Create a  Simple Service
+
+```ruby
+class SimpleService < Moleculer::Service::Base
+  action :get_user, :get_user
+  
+  
+  def get_user
+    # .. usery getting stuff
+  end
+end
+
+
+Moleculer.config do |c|
+  c.services << SimpleService
```

Comment:
Should we have something about the transporter setup here? 
Without the transporter it won't work will it?